wordpress-mobile/WordPress-iOS
# [Stats Refresh] Post Stats: fetch data for post

Ref #11189

This adds fetching of Post Stats data for the selected post. Since the data is not used yet, it is temporarily logged to the console.

Note: accompanying WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/147

To test:

- Access Post Stats via:
  - Insights > Latest Post Summary > View more
  - Period > Posts and Pages > row selection
  - Period > Posts and Pages > View more > row selection
- Verify the data appears in the console. It should look something like this: (Note `recentWeeks` has much more data than exampled here.)
```
🔴 postStats.monthlyBreakdown:  [WordPressKit.StatsPostViews(period: __C.StatsPeriodUnit, date: year: 2019 month: 4 isLeapMonth: false , viewsCount: 6730)]
🔴 postStats.dailyAveragesPerMonth:  [WordPressKit.StatsPostViews(period: __C.StatsPeriodUnit, date: year: 2019 month: 4 isLeapMonth: false , viewsCount: 799)]
🔴 postStats.recentWeeks:  [WordPressKit.StatsWeeklyBreakdown(startDay: year: 2019 month: 3 day: 11 isLeapMonth: false , endDay:  ...
```


- Pull to refresh. Verify the data appears in the console.
